packages/fxa-payments-server/public/locales/en-US/main.ftl

```diff
@@ -29,6 +29,9 @@ brand-name-firefox-logo = { -brand-name-firefox } logo
 close-aria =
   .aria-label = Close modal
 
+# Aria label for spinner image indicating data is loading
+app-loading-spinner-aria-label-loading = Loading...
```

Comment:
Not an issue on localization side, but pointing out this isn't defined as an attribute compared to other .aria-label strings.

Reply:
Note: This ID/message will be moved to fxa-react in issue #FXA-5998.

An alternative, from [draft PR-14440](https://github.com/mozilla/fxa/pull/14440):

```
# Aria label for loading spinner
app-loading-spinner-aria =
  .aria-label = Loading...
```

in component:
```
      <Localized id="app-loading-spinner-aria">
        <Spinner
          className="w-10 h-10 animate-spin"
          role="img"
          aria-label="Loading..."
        />
      </Localized>
```